awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -85,13 +84,16 @@ def yaml_dump(dict_to_dump):
     :param dict_to_dump:
     :return:
     """
-    FlattenAliasDumper.add_representer(OrderedDict, _dict_representer)
-    _add_yaml_1_1_boolean_resolvers(FlattenAliasDumper)
-    return yaml.dump(
-        dict_to_dump,
-        default_flow_style=False,
-        Dumper=FlattenAliasDumper,
-    )
+
+    yaml = ruamel.yaml.YAML(typ="safe", pure=True)
+    yaml.default_flow_style = False
+    yaml.representer.ignore_aliases = lambda *data: True
```

Comment:
I think I prefer if we just update the `FlattenAliasDumper` class to be a `FlattenAliasRepresenter` class instead of monkey patching the instance method with a lambda. It is a bit more explicit and easier to debug when there is a concrete class and method. We can then just update this line to be:
```python
yaml.Representer = FlattenAliasRepresenter
```

Reply:
Done